_data/ecosystem/ptc/2022/posters.yaml

```
@@ -0,0 +1,525 @@
+
+- authors:
+  - Justin Zhao
+
+  categories:
+  - LIBRARIES
+  description: "Ludwig is a declarative machine learning framework that makes it easy to define and compare machine learning pipelines using a simple and flexible data-driven configuration system. The minimal configuration declares the input and output features with their respective data types. Users can specify additional parameters to preprocess, encode, and decode features, load from pre-trained models, compose the internal model architecture, set training parameters, or run hyperparameter optimization. Ludwig will build an end-to-end machine learning pipeline automatically, using whatever is explicitly specified in the configuration, while falling back to smart defaults for any parameters that are not. Scientists, engineers, and researchers use Ludwig to explore state-of-the-art model architectures, run hyperparameter search, and scale up to larger than available memory datasets and multi-node clusters, on a variety of problems using structured and unstructured features. Ludwig has 8.5K+ stars on Github and is built on top of PyTorch, Horovod, and Ray."
+  link: 
+  poster_link:  https://pytorch.s3.amazonaws.com/posters/ptdd2022/B06.pdf
+  section: F8
+  thumbnail_link: https://pytorch.org/assets/images/ptc2022/B06-thumb.png   
+  title: "Declarative Machine Learning with Ludwig: End-to-end Machine Learning Pipelines Using Simple and Flexible Data-driven Configurations"
+
+- authors:
+  - Christian Puhrsch
+
+  categories:
+  - LIBRARIES
+  description: "This poster presents an overview of available and ongoing developments related to sparse memory formats, masked computation, and support for collections of variably shaped data. In particular it contains a case study of block sparse memory formats, MaskedTensor, and NestedTensor."
+  link: 
+  poster_link: https://pytorch.s3.amazonaws.com/posters/ptdd2022/B07.pdf 
+  section: F8
+  thumbnail_link: https://pytorch.org/assets/images/ptc2022/B07-thumb.png     
+  title: "Generalized Shapes: Block Sparsity, MaskedTensor, NestedTensor"
+
+- authors:
+  - Sang Keun Choe
+
+  categories:
+  - LIBRARIES
+  description: "Betty is a simple, scalable and modular library for generalized meta-learning (GML) and multilevel optimization (MLO), built upon PyTorch, that allows a unified programming interface for a number of GML/MLO applications including few-shot learning, hyperparameter optimization, neural architecture search, data reweighting, and many more. The internal autodiff mechanism and the software design of Betty are developed by the novel interpretation of GML/MLO as a dataflow graph."
+  link: 
+  poster_link: https://pytorch.s3.amazonaws.com/posters/ptdd2022/B08.pdf
+  section: F8
+  thumbnail_link: https://pytorch.org/assets/images/ptc2022/B08-thumb.png   
+  title: "Betty: An Automatic Differentiation Library for Generalized Meta Learning"
+
+
+
+- authors:
+  - Samantha Andow
+
```

Hmm, why not all authors are mentioned there?
```suggestion
  - Samantha Andow
  - Richard Zhou
  - Horace He
  - Animesh Jain
```